provision.sh

```diff
@@ -0,0 +1,28 @@
+# die on error
+set -e
+
+sudo apt-get update
+
+sudo apt-get -y install libatlas-base-dev libatlas-dev lib{blas,lapack}-dev gfortran
+sudo pip install conda
+conda_deps='pip numpy scipy'
+conda create -p $HOME/py --yes $conda_deps "python=$TRAVIS_PYTHON_VERSION"
+pip install -r requirements.txt
+python setup.py build_ext --inplace
+export PATH=$HOME/py/bin:$PATH
+
+echo 'Host github.com
```

Comment:
Why? I'm asking for my own education, as I'm not that familiar with the provisioning scripts.